### DISCUSSION TITLE
Community health and wellbeing check

### DISCUSSION THREAD

**Discussion by ghost** [studio-minus/ppg-bugs-and-requests#2375] [General]
### This is a community health check

write anything below (as long as it's civil and doesn't break any international laws)

![me when nobody replies](https://user-images.githubusercontent.com/104321411/201160502-20f32681-4cd0-4c9d-9ef6-5f7056bae569.jpg)

**Comment by ATPGaming** [comment]
Well, the game hasn't gotten updates in a long time. I feel like Zooi should review older suggestions because there are some in there that he could do easily, like making broken lightbulbs sharp. Small quality-of-life changes like this would renew interest in the game, even if it's a few new tweaks or a new item.

**Reply by ghost** [reply]
Yeah, I'm still waiting for the drill to get added.

**Reply by dinevolence** [reply]
I'm pretty sure it's because it's easier to just reject the bad features, instead of adding the good ones.

**Comment by Tbskins** [comment]
does anyone know if mestiez is doing good? I haven't heard anything from them in a while?

**Reply by Drogo-man** [reply]
Either having a mental break down, or taking a break.
